Snowdog_Components/components/Molecules/table/_table.scss

```diff
@@ -84,6 +84,25 @@
         }
     }
 
+    &--tfoot-border {
+        tfoot {
+            tr {
+                th {
+                    border-top: $table__tfoot-th-border-top\@medium;
+                    border-bottom: 0;
+
+                    &[scope="row"] {
+                        border-right-width: 1px;
```

Comment:
when working on styles in theme alpaca, we need to make sure styles can be easily modified in child projects. let's say in a child project we need to change all table widths to 2px. it would be great to be able to do it by overriding just one variable. I think we could make some improvements.
1. please create a new variable for border width. use `$border-width-base` as the value. it comes from file with global variables `Snowdog_Components/components/Atoms/variables/_variables.scss` 
2. use it in `$table__border`, `$table__th-border-width-row`, `$table__border-width--scope-row`
3. create a variable for this line of code